### ISSUE TITLE
Toast Quota

### ISSUE THREAD

**Issue by hkcowboy** [FossifyOrg/Messages#262]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Messages/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Messages/issues) or [closed](https://github.com/FossifyOrg/Messages/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Messages/discussions) or [closed](https://github.com/FossifyOrg/Messages/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Messages/blob/master/CONTRIBUTING.md).

### Affected app version

1.0.1

### Affected Android/Custom ROM version

crDroid 10.9

### Affected device model

Xiaomi 13 Ultra

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Open App and message shows immediately

### Expected behavior

No error

### Actual behavior

Toast Message saying `Error: org.xmlpull.v1.XmlPullParserException: unexpected type ...`

### Additional information

Here is the entry from MatLog: 
> `W/NotificationService(2251): Package org.fossify.messages is above allowed toast quota, the following toast was blocked and discarded: TextToastRecord{b2e0265 31790:org.fossify.messages/u0a532 isSystemToast=false token=android.os.BinderProxy@66ffb3a text=Error: org.xmlpull.v1.XmlPullParserException: unexpected type (position:END_DOCUMENT null@1:1 in java.io.StringReader@9872a6e)  duration=1}`

**Comment by Aga-C** [comment]
Have you received any MMS before this error started to show up? If so, what type of attachment does it have?

**Comment by hkcowboy** [comment]
Hello, this started as soon as I installed the app and it keeps popping up whenever I open the app and anytime i interact with the app (e.g., click back button, click on search bar, open conversation).  I am able to send and receive messages: text & image, so I am not sure what is going on.

**Comment by hkcowboy** [comment]
Hi, what can be done about the error?

**Comment by Aga-C** [comment]
Until we investigate it further, it's hard to tell what can be done to avoid it. It's probably related to some MMS messages that may be corrupted, but I'm not sure.

**Comment by naveensingh** [comment]
Yeah, it's definitely an MMS message the app is failing to parse. 

This could also happen while trying to import messages but that's probably not the case here.

**Comment by hkcowboy** [comment]
How would I identify which MMS message is the problem?


> Yeah, it's definitely an MMS message the app is failing to parse.
> 
> This could also happen while trying to import messages but that's probably not the case here.

**Comment by naveensingh** [comment]
> How would I identify which MMS message is the problem? Maybe I can delete it so the parsing issue stops

Depends. Can you make some code changes and compile an app? 

**If yes:**

 - Catch the exception here:     https://github.com/FossifyOrg/Messages/blob/58169dfd2c3a978cd07b0635bfee2d237f8d2733/app/src/main/kotlin/org/fossify/messages/extensions/Context.kt#L505-L506
 - Log the attachment text or delete the MMS directly
 - Build and install the app


**If not:**

It'll be fixed soon-ish.

**Comment by naveensingh** [comment]
Should be fixed in the next update.

**Comment by hkcowboy** [comment]
Hi, just want to say that the issue has been resolved for me with the new update. Thank you very much!
